cmd/swctl/app/cmd_config.go

```diff
@@ -57,3 +85,34 @@ func runConfigCmd(cli Cli, opts ConfigCmdOptions) error {
 	fmt.Fprintln(cli.Out(), stdout)
 	return nil
 }
+
+func mergeHelpers(cmd *cobra.Command, ParsedFlags []temporaryFlag, cli Cli, args []string) (string, string, bool, error) {
```

Comment:
`ParsedFlags` should be `parsedFlags` as it is not exported. Same applies to usage of `ParsedFlags` in the `runConfigCmd` function above.

Reply:
Another thing: is the `ParsedFlags` argument even needed? In the end it is only assigned to _.